op-e2e/celo/tests/setup.js

```diff
@@ -0,0 +1,60 @@
+import { setupClients } from '../src/config.js'
+import { makeChainConfigs } from '../src/chain.js'
+import { privateKeyToAccount } from 'viem/accounts'
+import { readFileSync } from 'fs'
+
+const privKey =
+  '0x92db14e403b83dfe3df233f83dfa3a0d7096f21ca9b0d6d6b8d88b2b4ec1564e'
```

Comment:
Do we want to have private key in repo ?

Reply:
That's a default Anvil dev account (mnemonic `test test test test test test test test test test test junk` ) that gets a pre allocation in the devnet

Reply:
I'll add a comment above the definition